connection-cache/src/connection_cache.rs

```diff
@@ -18,38 +19,56 @@ use {
 // Should be non-zero
 pub static MAX_CONNECTIONS: usize = 1024;
 
-/// Used to decide whether the TPU and underlying connection cache should use
-/// QUIC connections.
-pub const DEFAULT_TPU_USE_QUIC: bool = true;
+/// Default connection pool size per remote address
+pub const DEFAULT_CONNECTION_POOL_SIZE: usize = 4;
```

Comment:
Do we need this const as well as `tpu_client::tpu_client::DEFAULT_TPU_CONNECTION_POOL_SIZE`?

Reply:
I think we can keep both. It is possible for TPU and other workflows uses different default values.